mlflow/server/js/package.json

```diff
@@ -83,13 +79,16 @@
       "lcov"
     ]
   },
-  "proxy": {
-    "/ajax-api": {
-      "target": "http://localhost:5000"
-    },
-    "/get-artifact": {
-      "target": "http://localhost:5000",
-      "ws": true
-    }
+  "browserslist": {
```

Comment:
Could you please provide more information on why we need this?

Reply:
I am not sure about why it was not included in the first place. 
But when I tried to build the package with react-scripts v3 it was mandatory.

https://create-react-app.dev/docs/supported-browsers-features#configuring-supported-browsers